backend/internal/models/event.go

```diff
@@ -0,0 +1,22 @@
+package models
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+// Represents Venue Event
+type Event struct {
+	EventID   int       `json:"event_id"`
+	Name      string    `json:"name"`
+	Date      time.Time `json:"event_date"`

```

Comment:
having an `Event.Date` of type `time.Time` and an `Event.Time` of type `time.Time` is redundant